frontend/src/app/features/readers/ebook-reader/layout/footer/footer.component.scss

```
@@ -413,11 +413,77 @@
+    }
+
+    .progress-section {
+      gap: 10px;
+
+      .location-btn {
+        transform: translateX(-0.5rem);
+      }
+
+      .progress-slider-wrapper {
+        
+      }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)